src/main/java/camp/nextstep/edu/rebellion/lotto/domain/ticket/LottoNumber.java

```diff
@@ -1,27 +1,46 @@
 package camp.nextstep.edu.rebellion.lotto.domain.ticket;
 
+import camp.nextstep.edu.rebellion.lotto.rule.LottoGameRule;
+
 public class LottoNumber {
-    private static final int LOTTO_MIN_NUMBER = 1;
-    private static final int LOTTO_MAX_NUMBER = 45;
-    private final int number;
+    private final Integer number;
 
     public LottoNumber(int number) {
         checkRange(number);
-        this.number = number;
+        this.number = Integer.valueOf(number);
+    }
+
+    public LottoNumber(String str) {
+        this(Integer.parseInt(str));
     }
 
-    public boolean equals(LottoNumber number) {
-        return this.number == number.getNumber();
+    public boolean equals(LottoNumber n) {
+        return this.number == n.number;
     }
```

Comment:
38~45 라인과 동일한 기능을 할 것 같아요.

Reply:
`It's almost always a mistake to compare two instances of java.lang.String or boxed types like java.lang.Integer using reference equality == or !=, because it is not comparing actual value but locations in memory.`

애초에 박싱타입을 선택하신 이유도 궁금하네요.

Reply:
엄청 중요한 부분을 잘 잡아주셔서 감사합니다 :-) 아무 생각 없이 하던 습관을 이번에 고쳐야겠습니다 
* https://rules.sonarsource.com/java/RSPEC-4973
* https://wiki.sei.cmu.edu/confluence/display/java/EXP03-J.+Do+not+use+the+equality+operators+when+comparing+values+of+boxed+primitives